samples/v1.5/Scenarios/FlightUpdateTable.json

```diff
@@ -31,7 +31,7 @@
 							"items": [
```

Comment:
Can we also update the corresponding template sample?

Reply:
okay